Code/tests/test_chocolatey_packager.py

```diff
@@ -37,10 +37,16 @@ def get_mocked_writes(mock: unittest.mock.MagicMock) -> str:
     return res
 
 
+def check_for_choco() -> bool:
+    try:
+        return subprocess.run(["choco.exe", "--version"]).returncode == 0
```

Comment:
This is generally going to be fine, but I greatly prefer using an absolute path. Choco installs in a fixed path (I think?), and trusting the powershell path makes me itchy.

Reply:
It does appear to have a fixed installation path. I'll change this.